local/scripts/dist.sh

```diff
@@ -33,6 +33,8 @@ git commit --allow-empty --message "$COMMIT_MESSAGE"
 if [ -n "$DIST_TAG" ]; then
 	echo "Tagging a release: $DIST_TAG"
 	git tag --force "$DIST_TAG"
+	git push --force --set-upstream origin --tags
```

Comment:
@dero I think the issue was caused by the `$DIST_BRANCH` being used together with the `--tags` flag. Now it will force-push all tags or just the specific branch.

I've tested this locally and it appears to be working as expected. Since this particular file isn't included in the release bundle I suggest we merge this in and test during the next release.

Reply:
Yes, looks good now. :+1: